crates/bevy_gizmos/src/config.rs

```diff
@@ -36,14 +37,40 @@ pub enum GizmoLineJoint {
 }
 
 /// An enum used to configure the style of gizmo lines, similar to CSS line-style
-#[derive(Copy, Clone, Debug, Default, Hash, PartialEq, Eq, Reflect)]
+#[derive(Copy, Clone, Debug, Default, PartialEq, Reflect)]
 #[non_exhaustive]
 pub enum GizmoLineStyle {
     /// A solid line without any decorators
     #[default]
     Solid,
     /// A dotted line
     Dotted,
+    /// A dashed line with configurable gap and line sizes
+    Dashed {
+        /// The length of the gap in `line_width`s
+        gap_scale: f32,
+        /// The length of the visible line in `line_width`s
+        line_scale: f32,
+    },
+}
+impl Eq for GizmoLineStyle {}
+impl Hash for GizmoLineStyle {
```

Comment:
Let's add newlines for consistency:
```suggestion

impl Eq for GizmoLineStyle {}

impl Hash for GizmoLineStyle {
```